src/pybind/mgr/cephadm/module.py

```diff
@@ -3054,6 +3054,16 @@ def set_prometheus_access_info(self, user: str, password: str) -> str:
         self.set_store(PrometheusService.USER_CFG_KEY, user)
         self.set_store(PrometheusService.PASS_CFG_KEY, password)
         return 'prometheus credentials updated correctly'
+    
+    @handle_orch_error
+    def set_prometheus_cert(self, cert: str) -> str:
+        self.set_store(PrometheusService.PROMETHEUS_CERT_CFG_KEY, cert)
+        return 'prometheus cert stored correctly'
```

Comment:
Note, I'm going to end up changing this as part of https://github.com/ceph/ceph/pull/55564 (assuming this gets merged first)

Reply:
Okay, sounds good